_data/meltano/extractors/tap-s3-csv/s7clarke10.yml

```
@@ -0,0 +1,92 @@
+capabilities:
+- properties
+- discover
+- state
+description: Extract CSV files from S3
+domain_url: https://aws.amazon.com/s3/
+keywords:
+- api
+label: AWS S3 CSV
+logo_url: /assets/logos/extractors/s3-csv.png
+maintenance_status: active
+name: tap-s3-csv
+namespace: tap_s3_csv
+pip_url: pipelinewise-tap-s3-csv
```

```suggestion
pip_url: git+https://github.com/s7clarke10/pipelinewise-tap-s3-csv.git
```

Should this be referencing your repo?